linux_proc_extras/datadog_checks/linux_proc_extras/linux_proc_extras.py

```diff
@@ -99,3 +101,15 @@ def get_process_states(self):
             prio_tags = list(self.tags)
             prio_tags.append("priority:" + prio)
             self.gauge('system.processes.priorities', float(prio_counts[prio]), prio_tags)
+
+    def get_interrupts_info(self):
+        with open(self.proc_path_map['interrupts_info'], 'r') as interrupts_info:
+            lines = [line.strip() for line in interrupts_info.readlines()]
+            cpu_count = len(lines[0].split())
+            for line in lines[1:]:
+                parts = line.split()
+                irq_id = parts[0].replace(":", "")
+                for cpu_id, part in enumerate(parts[1 : cpu_count + 1]):
+                    irq_count = int(part)
+                    tags = self.tags + ['irq:{}'.format(irq_id), 'cpu_id:{}'.format(cpu_id)]
+                    self.monotonic_count("system.linux.interrupts.stats", irq_count, tags=tags)
```

Comment:
TODO: Find a better name if possible. `system.linux.interrupts` is already taken to represent the total number of interrupts.

Reply:
Renamed to `irq`